.github/workflows/nodejs.yml

```
@@ -19,10 +21,10 @@ jobs:
       fail-fast: false
       matrix:
         os: [macos-latest, windows-latest, ubuntu-latest]
-        node-version: [14, 16]
+        node-version: [14, 16, 18]
```

might want to use the relative notation here if you want automatic CI upgrades - totally understandable if not!

```suggestion
        node-version: [current, lts/*, lts/-1, lts/-2]
```